MediaAdminBundle/Resources/public/less/media.less

```diff
@@ -1,112 +1,122 @@
 .figure {
-  vertical-align: top;
-  position: relative;
-  height: 127px;
-  width: 127px;
-}
-.figure .img-container{
-  line-height: 112px;
-  text-align: center;
-  background: slategrey;
-  height: 100%;
-}
-.figure .img-container img{
-  vertical-align:middle;
-  max-width: 117px;
-  max-height: 117px;
-}
+    vertical-align: top;
```

Comment:
as it affects the form of alternative generation and image display, I take advantage to do.